### ISSUE TITLE
how to install alpha chess zero

### ISSUE THREAD

**Issue by geo17007** [Zeta36/chess-alpha-zero#46]
can anyone make a video of how to install alpha zero to arena??

**Comment by Akababa** [comment]
Do the instructions in readme not work?

**Comment by geo17007** [comment]
nope... it's a little bit confusing! i much appreciate if you do a video or step by step guide.ty for your response

**Comment by Akababa** [comment]
It should be really simple, just make a new engine in Arena and browse to C0uci.bat where you cloned the repo. (rename it to .sh if you're on linux)

If that still doesn't work can you open the debug window in Arena (F4) and show it here? Thanks

**Comment by StormXploit** [comment]
For me it doesn't work to.  I download everything.  I rename COuci becuz I'm on Linux.  I exe the script but it doesn't work.  Help me too pks

**Comment by StormXploit** [comment]
My terminal write : 
Traceback (most recent call)  
File src/chess_zero/run. Py,  line 13 in module
MP set start method (,spawn) 
attribute error : Mosul object has not attribute start method

**Comment by geo17007** [comment]
here is my debug window in arena

2018-01-05 15:53:00,224 Arena 3.5.1
2018-01-05 15:53:00,232
2018-01-05 15:53:00,716**----------New game---2018-01-05 15:53:00,716 Ðáñ -------------
2018-01-05 15:53:00,848**screen: 1920x1080
2018-01-05 15:53:00,848**Monitors: 1
2018-01-05 15:53:00,848**Monitor0: 1920x1080
2018-01-05 15:53:00,848**FormMonitor: 0
2018-01-05 15:53:00,848**DesktopRect: 0/0 .. 1920/1080
2018-01-05 15:53:01,006**Loading  1
2018-01-05 15:53:52,685*1*-----------------------Starting engine 1 C0uci [006]------------------------
2018-01-05 15:53:52,688*1*Configured Engine 1 Type:  Auto
2018-01-05 15:53:52,688*1*Engine 1 dir: C:\Users\Geo\Desktop\Alpha Zero chess\chess-alpha-zero-master\chess-alpha-zero-master
2018-01-05 15:53:52,688*1*Engine 1 commandline: "C:\Users\Geo\Desktop\Alpha Zero chess\chess-alpha-zero-master\chess-alpha-zero-master\C0uci.bat" 
2018-01-05 15:53:52,800*1*Engine 1 ProcessID: 3548
2018-01-05 15:53:52,800*1*Engine 1 Prio:32 ThreadPrio:0
2018-01-05 15:53:52,800<--1:C:\Users\Geo\Desktop\Alpha Zero chess\chess-alpha-zero-master\chess-alpha-zero-master>python src/chess_zero/run.py uci 
2018-01-05 15:53:52,801<--1:'python' is not recognized as an internal or external command,
2018-01-05 15:53:52,801<--1:operable program or batch file.
2018-01-05 15:53:52,801-->1:xboard
2018-01-05 15:53:52,822-->1:uci
2018-01-05 15:54:00,828-->1:isready
2018-01-05 15:54:01,503-->1:new
2018-01-05 15:54:01,503-->1:random
2018-01-05 15:54:01,503-->1:level 0 11 0
2018-01-05 15:54:01,503-->1:post
2018-01-05 15:54:01,503-->1:hard
2018-01-05 15:54:01,503-->1:easy
2018-01-05 15:54:24,264-->1:computer
2018-01-05 15:54:24,295*1*Incorrect function
2018-01-05 15:54:24,295*1*Engine crashed, restarting...
2018-01-05 15:54:24,295-->1:?
2018-01-05 15:54:24,295-->1:force
2018-01-05 15:54:24,295-->1:quit
2018-01-05 15:54:24,653*1*-----------------------Starting engine 1 C0uci [006]------------------------
2018-01-05 15:54:24,653*1*Configured Engine 1 Type:  Auto
2018-01-05 15:54:24,653*1*Engine 1 dir: C:\Users\Geo\Desktop\Alpha Zero chess\chess-alpha-zero-master\chess-alpha-zero-master
2018-01-05 15:54:24,653*1*Engine 1 commandline: "C:\Users\Geo\Desktop\Alpha Zero chess\chess-alpha-zero-master\chess-alpha-zero-master\C0uci.bat" 
2018-01-05 15:54:24,760*1*Engine 1 ProcessID: 5784
2018-01-05 15:54:24,760*1*Engine 1 Prio:32 ThreadPrio:0
2018-01-05 15:54:24,764<--1:C:\Users\Geo\Desktop\Alpha Zero chess\chess-alpha-zero-master\chess-alpha-zero-master>python src/chess_zero/run.py uci 
2018-01-05 15:54:24,764<--1:'python' is not recognized as an internal or external command,
2018-01-05 15:54:24,856<--1:operable program or batch file.
2018-01-05 15:54:24,860-->1:xboard
2018-01-05 15:54:24,880-->1:uci
2018-01-05 15:55:05,390-->1:isready
2018-01-05 15:55:06,075-->1:new
2018-01-05 15:55:06,075-->1:random
2018-01-05 15:55:06,075-->1:level 0 11 0
2018-01-05 15:55:06,075-->1:post
2018-01-05 15:55:06,075-->1:hard
2018-01-05 15:55:06,075-->1:easy
2018-01-05 15:55:06,076-->1:computer
2018-01-05 15:55:06,926*1*Start calc, move no: 1
2018-01-05 15:55:06,984-->1:level 0 11 0
2018-01-05 15:55:06,984-->1:new
2018-01-05 15:55:06,985-->1:random
2018-01-05 15:55:06,985-->1:level 0 11 0
2018-01-05 15:55:06,986-->1:post
2018-01-05 15:55:06,986-->1:hard
2018-01-05 15:55:06,987-->1:easy
2018-01-05 15:55:06,988-->1:computer
2018-01-05 15:55:06,989-->1:time 66000
2018-01-05 15:55:06,990-->1:otim 66000
2018-01-05 15:55:06,990-->1:e2e4

**Comment by StormXploit** [comment]
Ok but i'm on linux

**Comment by brianprichardson** [comment]
xboard is a CECP command, not UCI.  Make sure you have specified UCI for the engines you want to run.  Also, running a Python engine under Arena is a separate and tricky  issue, so the xboard command may be triggered by not processing the python command first, and then Arena defaulting to CECP.  Suggest searching on talkchess for posts about running Python engines.

**Comment by geo17007** [comment]
ty for your responde brian but i think i just try to run that engine like others....not with python..!i think i have to study a lot...

**Comment by Akababa** [comment]
That's weird how it says python isn't found... can you run `python -V`?

**Comment by StormXploit** [comment]
Yes

**Comment by StormXploit** [comment]
Very strange

**Comment by StormXploit** [comment]
Can you create a discord server ? The interaction will be easier if think

**Comment by Akababa** [comment]
Ok. IDK how often I will check discord though

https://discord.gg/myWbdcD

**Comment by brianprichardson** [comment]
@geo17007 You might look here to just run everything under Python:
[https://github.com/niklasf/python-chess](url)

**Comment by StormXploit** [comment]
I've installed chess arena but i've always the same problem. Someone can help me to resolve it please

**Comment by geo17007** [comment]
your link brian isnt worked...404 not found

**Comment by brianprichardson** [comment]
I have had trouble posting links before.
Copy it and paste it and it should work, or just google it.

**Comment by aksdeo5** [comment]
Anyone successfully implemented this program into any UCI Chess GUI...?!
@Akababa : Is it possible to implement this program in WinBoard using polyglot uci adapter...?! If it is possible, then please provide step by step procedure to do it. It'll be better if you make video and provide the link of the video. Thanks in advance!!
Regards...
